utils/wasm-builder/src/lib.rs

```diff
@@ -79,6 +75,18 @@ impl WasmBuilder {
         }
     }
 
+    /// Build the program and produce an output WASM binary.
+    pub fn build(self) {
+        if env::var(self.cargo.skip_package_build_env()).is_ok()
+            || env::var("SKIP_WASM_BUILD").is_ok()
+        {
+            Self::exit(self.wasm_project.write_wasm_binary_file(None));
```

Comment:
Add todo to consider option of avoid writing empty file in favour of already existent one